### ISSUE TITLE
Crash with 0.2.1

### ISSUE THREAD

**Issue by yochananmarqos** [johanthoren/bibcal#1]
`bibcal-bin` 0.1.1 was working fine.

Manjaro GNOME (unstable branch)

```
Exception in thread "main" Syntax error compiling fn* at (doric/org.clj:1:1).
	at clojure.lang.Compiler.analyzeSeq(Compiler.java:7119)
	at clojure.lang.Compiler.analyze(Compiler.java:6793)
	at clojure.lang.Compiler.eval(Compiler.java:7178)
	at clojure.lang.Compiler.eval(Compiler.java:7170)
	at clojure.lang.Compiler.load(Compiler.java:7640)
	at clojure.lang.RT.loadResourceScript(RT.java:381)
	at clojure.lang.RT.loadResourceScript(RT.java:372)
	at clojure.lang.RT.load(RT.java:459)
	at clojure.lang.RT.load(RT.java:424)
	at clojure.core$load$fn__6856.invoke(core.clj:6115)
	at clojure.core$load.invokeStatic(core.clj:6114)
	at clojure.core$load.doInvoke(core.clj:6098)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at clojure.core$load_one.invokeStatic(core.clj:5897)
	at clojure.core$load_one.invoke(core.clj:5892)
	at clojure.core$load_lib$fn__6796.invoke(core.clj:5937)
	at clojure.core$load_lib.invokeStatic(core.clj:5936)
	at clojure.core$load_lib.doInvoke(core.clj:5917)
	at clojure.lang.RestFn.applyTo(RestFn.java:142)
	at clojure.core$apply.invokeStatic(core.clj:669)
	at clojure.core$load_libs.invokeStatic(core.clj:5974)
	at clojure.core$load_libs.doInvoke(core.clj:5958)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:669)
	at clojure.core$require.invokeStatic(core.clj:5996)
	at clojure.core$require.doInvoke(core.clj:5996)
	at clojure.lang.RestFn.invoke(RestFn.java:408)
	at doric.core$table_STAR_.invokeStatic(core.clj:158)
	at doric.core$table_STAR_.doInvoke(core.clj:143)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.core$apply.invokeStatic(core.clj:667)
	at clojure.core$apply.invoke(core.clj:662)
	at doric.core$table.invokeStatic(core.clj:173)
	at doric.core$table.doInvoke(core.clj:167)
	at clojure.lang.RestFn.invoke(RestFn.java:421)
	at xyz.thoren.bibcal$print_date.invokeStatic(bibcal.clj:122)
	at xyz.thoren.bibcal$print_date.invoke(bibcal.clj:116)
	at xyz.thoren.bibcal$_main.invokeStatic(bibcal.clj:243)
	at xyz.thoren.bibcal$_main.doInvoke(bibcal.clj:225)
	at clojure.lang.RestFn.invoke(RestFn.java:397)
	at clojure.lang.AFn.applyToHelper(AFn.java:152)
	at clojure.lang.RestFn.applyTo(RestFn.java:132)
	at xyz.thoren.bibcal.main(Unknown Source)
Caused by: com.oracle.svm.core.jdk.UnsupportedFeatureError: Defining a class from new bytecodes at run time is not supported. Class clojure.core$eval146 with hash d096b8b0e96c1c19d33456bb8e3a40b9ee6c3cf1be20a354c10538d775fd36c7 was not provided during the image build. Please see BuildConfiguration.md.
	at com.oracle.svm.core.util.VMError.unsupportedFeature(VMError.java:88)
	at com.oracle.svm.core.hub.PredefinedClassesSupport.loadClass(PredefinedClassesSupport.java:93)
	at java.lang.ClassLoader.defineClass(ClassLoader.java:329)
	at clojure.lang.DynamicClassLoader.defineClass(DynamicClassLoader.java:46)
	at clojure.lang.Compiler$ObjExpr.getCompiledClass(Compiler.java:4990)
	at clojure.lang.Compiler$FnExpr.parse(Compiler.java:4120)
	at clojure.
```

**Comment by johanthoren** [comment]
@yochananmarqos, thanks for reporting this. I'm going to spend some time today adding integration tests of the freshly built binaries, so that this doesn't happen again. I'll look into the problem once tests are in place.

**Comment by johanthoren** [comment]
This has been solved in version 0.2.2. One of the libraries I imported, https://github.com/joegallo/doric, was working fine with the JVM but didn't play ball when compiled into a graal-vm native-image. This library has been replaced for now.
